PMDMMOc/src/com/darkxell/client/mechanics/event/ClientEventProcessor.java

```diff
@@ -202,8 +191,6 @@ private void doClientProcess(DungeonEvent event) {
             this.processItemEvent((ItemSelectionEvent) event);
         if (event instanceof ItemMovedEvent)
             this.processItemMovedEvent((ItemMovedEvent) event);
-        if (event instanceof ItemSwappedEvent)
```

Comment:
Really minor but important quibble, could you please make your commits more atomic when it makes sense? It really helps me understand why a change was made.

Also applies to when you update javadoc formatting.